### PR TITLE
fix: distribute /tools/rust_analyzer

### PR DESCRIPTION
**Pull request by roman-kashitsyn**
It seems that https://github.com/bazelbuild/rules_rust/pull/1194 accidentally excluded //tools/rust_analyzer from the distribution.
This change adds rust_analyzer back.

### CODE REVIEW

**Review by UebelAndre — APPROVED** [review]
Thank you!

**Comment by UebelAndre** [comment]
@roman-kashitsyn once the CLA is satisfied I can get this merged.

**Comment by roman-kashitsyn** [comment]
> @roman-kashitsyn once the CLA is satisfied I can get this merged.

Yes, sorry for the delay, I'm waiting for confirmation from my employer.

**Pull request closed without merging**
